Libraries/Opc.Ua.Configuration/ApplicationConfigurationBuilder.cs

```diff
@@ -141,19 +141,95 @@ public IApplicationConfigurationBuilderClientSelected AsClient()
                     StoreType = rejectedRootType,
                     StorePath = DefaultCertificateStorePath(TrustlistType.Rejected, rejectedRoot)
                 },
-                // ensure secure default settings
-                AutoAcceptUntrustedCertificates = false,
-                AddAppCertToTrustedStore = false,
-                RejectSHA1SignedCertificates = true,
-                RejectUnknownRevocationStatus = true,
-                SuppressNonceValidationErrors = false,
-                SendCertificateChain = true,
-                MinimumCertificateKeySize = CertificateFactory.DefaultKeySize
             };
+            SetSecureDefaults(ApplicationConfiguration.SecurityConfiguration);
 
             return this;
         }
 
+        /// <inheritdoc/>
+        public IApplicationConfigurationBuilderSecurityOptionStores AddSecurityConfigurationStores(
+            string subjectName,
+            string appRoot,
+            string trustedRoot,
+            string issuerRoot,
+            string rejectedRoot = null
```

Comment:
Why does only `rejectedRoot` have default ?

Reply:
Trying to support a minimal config, e.g. in a pod there is no point to save the rejected certs if they cannot be accessed

Reply:
OK, understood.